cyclient/models.py

```diff
@@ -104,6 +104,71 @@ def build_dto(self, data, **kwargs):
         return ScanResult(**data)
 
 
+class ScanPollingResult(Schema):
+    def __init__(self, scan_id: str = None, err: str = None):
+        super().__init__()
+        self.scan_id = scan_id
+        self.err = err
```

Comment:
error

Reply:
here we are using err - ZippedFileScanResult